src/main/java/edu/hm/hafner/analysis/Report.java

```diff
@@ -53,12 +58,12 @@
 @SuppressWarnings({"PMD.ExcessivePublicCount", "PMD.ExcessiveClassLength", "PMD.GodClass"})
 // TODO: provide a readResolve method to check the instance and improve the performance (TreeString, etc.)
 public class Report implements Iterable<Issue>, Serializable {
-    private static final long serialVersionUID = 2L; // release 8.0.0
+    private static final long serialVersionUID = 3L; // release 8.0.
 
     @VisibleForTesting
     static final String DEFAULT_ID = "-";
 
-    private final Set<Issue> elements = new LinkedHashSet<>();
+    private transient Set<Issue> elements = new LinkedHashSet<>();
```

Comment:
Why is this transient? Shouldn't it just be not final? Otherwise Jenkins XML-Serialization will break.